### PR TITLE
Resend activation email for new users as admin

### LINKED ISSUE #6023: Option to resend activation email for new users as admin
This is probably a new feature request in the section "users and groups" and "activity and notification"

**Status quo**
- Checkbox «Send email to new user» activated
- Admin creates new user and activation email was sent
- Password reset link (token) expired before user was able to set his new password (within 24 hours?)
- Admins cannot resend the activation email via user administration
- Admins have to delete and recreate a user or set a password and inform the user manually

**Requested features**
- Possibility for admins and group admins to resend activation emails with a link to set the password (fresh token)
- _Optional: Change lifetime of reset password links/tokens maybe as config.php parameter_

### PR DESCRIPTION
**Pull request by skjnldsv**
Fix #6023 

@nextcloud/designers @jonasgarstick

### CODE REVIEW

**Review by MorrisJobke — APPROVED** [review]
Tested and works 👍

**Comment by MorrisJobke** [comment]
Rebase to fix the conflicts ;)

**Comment by JK74** [comment]
Gents, great to hear!

 what about extending the timeout for activation from 24h to 48 or 96 or longer?

**Comment by skjnldsv** [comment]
@JK74 it's not the same issue though. Could you check if there is not already an issue on this?

**Comment by JK74** [comment]
Yes will do but it's an alternative solution as I think most people like to see the resolution in 2 ways: manual re-submission and extensing the activation time

File: settings/js/settings-vue.js [hunk 1]
```diff
@@ -40,7 +40,11 @@ var n="undefined"!=typeof window&&"undefined"!=typeof document,r=["Edge","Triden
   * vue-router v3.0.1
   * (c) 2017 Evan You
   * @license MIT
+<<<<<<< HEAD
```
Comment (rullzer) [inline settings/js/settings-vue.js]: This doesn't look right

Comment (skjnldsv) [reply settings/js/settings-vue.js]: Indeed :see_no_evil:

Comment (skjnldsv) [reply settings/js/settings-vue.js]: Done

**Review by rullzer — APPROVED** [review]

**Pull request merged**